app/models/school.rb

```diff
@@ -43,6 +45,14 @@ def partially_registered?
       &.any?
   end
 
+  def local_authority
+    school_local_authorities.find_by(end_year: nil)&.local_authority
```

Comment:
I don't quite get why we need end_year to be nil.

Reply:
Cause that's how we find the latest one - without any end time (I think)

Reply:
Because they have not left that local authority. End year is the year a school is no longer a member of that local authority

Reply:
We could introduce a friendly scope so that it would be easier to read - scope :latest could do onschool_local_authority

Reply:
For a scope it would have to be `where(end_year: nil)`, which we would need to call `.first` on. Is it worth it?